packages/big-design/src/components/ProgressCircle/ProgressCircle.tsx

```diff
@@ -10,26 +12,10 @@ export interface ProgressCircleProps {
   size?: 'xSmall' | 'small' | 'medium' | 'large';
 }
 
-export class ProgressCircle extends React.PureComponent<ProgressCircleProps> {
-  static readonly defaultProps: Partial<ProgressCircleProps> = {
-    size: 'medium',
-  };
-
-  render() {
-    const { error } = this.props;
-
-    return error ? this.renderError() : this.renderCircle();
-  }
-
-  getDimensions() {
-    const { size } = this.props;
-
-    return size ? CIRCLE_DIMENSIONS[size] : CIRCLE_DIMENSIONS.medium;
-  }
-
-  renderCircle() {
-    const { percent, size } = this.props;
+export const ProgressCircle: React.FC<ProgressCircleProps> = typedMemo(({ error, percent, size = 'medium' }) => {
+  const getDimensions = () => (size ? CIRCLE_DIMENSIONS[size] : CIRCLE_DIMENSIONS.medium);
```

Comment:
I think it's a redundant condition because by default `size` equal to `medium`. So it could just `CIRCLE_DIMENSIONS[size]`